interfaces/acados_template/acados_template/acados_ocp_batch_solver.py

```diff
@@ -111,14 +124,19 @@ def ocp_solvers(self):
     def N_batch(self):
         """Batch size."""
         return self.__N_batch
+    
+    @property
+    def num_threads_in_batch_solve(self):
+        """Number of threads used for parallelizing the batch methods."""
+        return self.__num_threads_in_batch_solve
```

Comment:
It seems like the setter is missing to actually make it dynamic.
Can you add it and also test it in the example?